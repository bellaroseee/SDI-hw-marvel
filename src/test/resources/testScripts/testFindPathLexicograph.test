CreateGraph graph1

AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D

AddEdge graph1 A B 1.0
AddEdge graph1 B C 1.0
AddEdge graph1 A D 4.0
AddEdge graph1 C D 1.0

FindPath graph1 A D

FindPath graph1 A C