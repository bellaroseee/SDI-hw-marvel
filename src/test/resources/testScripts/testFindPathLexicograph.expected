created graph graph1

added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1

added edge 1.0 from A to B in graph1
added edge 1.0 from B to C in graph1
added edge 4.0 from A to D in graph1
added edge 1.0 from C to D in graph1

path from A to D:
A to D via 4.0

path from A to C:
A to B via 1.0
B to C via 1.0